.gitignore

```diff
@@ -0,0 +1,4 @@
+templator.sh
+template*
+temp*
+spkg*
```

Comment:
We should not include this .gitignore.  We may exclude things we might want to include such as a package named temperature.rb, for example.  Most commits in this repo are packages and a `.gitignore` is better off ... ignored. :)

Reply:
I plan to remove that after I get everything working!